src/VAC/View.cpp

```diff
@@ -240,24 +246,24 @@ int View::decideClicAction()
         }
 
         // Cut edge
-        if( global()->toolMode() == Global::SELECT &&
-             mouse_LeftButton_ &&
-             !mouse_AltWasDown_ &&
-             mouse_ControlWasDown_ &&
-             !mouse_ShiftWasDown_)
-        {
-            return SPLIT_ACTION;
-        }
+//        if( global()->toolMode() == Global::SELECT &&
+//             mouse_LeftButton_ &&
+//             !mouse_AltWasDown_ &&
+//             mouse_ControlWasDown_ &&
+//             !mouse_ShiftWasDown_)
+//        {
+//            return SPLIT_ACTION;
+//        }
```

Comment:
add comment why this is commented out, or if it's not needed in future, remove it

Reply:
removed